src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Locale.xml

```diff
@@ -16,7 +16,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
 >
-    <resource class="Sylius\Component\Locale\Model\Locale" shortName="Locale">
+    <resource class="Sylius\Component\Locale\Model\Locale" shortName="Locale" paginationEnabled="false">
```

Comment:
Why? The best guess I have is it was added to show all available locales on the shop's side at once, but even so, this should be applied to the operation not to the whole resource.

Reply:
Moved to the get collection endpoint 👍🏻